model/project_ref.go

```diff
@@ -1921,6 +1923,15 @@ func FindProjectRefs(key string, limit int, sortDir int) ([]ProjectRef, error) {
 	return projectRefs, err
 }
 
+// CanEnableRepotracker checks if the project can enable the repotracker
+// with the current state of their project settings.
+func (p *ProjectRef) ValidateModifiedRepotracker() error {
+	if p.Enabled && p.RemotePath == "" {
+		return errors.Errorf("Your project '%s' must have a confguration set to enable repotracker", p.Identifier)
```

Comment:
i think it's not quite right to return an error here as though it's written to the user, since it's a helper function. I'd suggest just "remote path can't be empty for enabled repotracker project" or something (it's possible that they didn't just enable the repotracker or something for example, they may have just enabled the project or just removed the config, so this comment might be confusing)